content/v2.0/reference/api/influxdb-1x/dbrp.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Database and retention policy mapping
+description: >
+  The database and retention policy (DBRP) mapping service maps InfluxDB 1.x
+  database and retention policy combinations to InfluxDB 2.0 buckets.
+menu:
+  v2_0_ref:
+    name: DBRP mapping
+    parent: 1.x compatibility
+weight: 302
+products: [cloud]
+related:
+  - /v2.0/reference/api/influxdb-1x/query
+  - /v2.0/reference/api/influxdb-1x/write
+  - /v2.0/api/#tag/DBRPs, InfluxDB 2.0 API /dbrps endpoint
+---
+
+The InfluxDB 1.x data model includes [databases](https://docs.influxdata.com/influxdb/v1.8/concepts/glossary/#database)
+and [retention policies](https://docs.influxdata.com/influxdb/v1.8/concepts/glossary/#retention-policy-rp).
+InfluxDB 2.0 replaces both with [buckets](/v2.0/reference/glossary/#bucket).
+To support InfluxDB 1.x query and write patterns in InfluxDB 2.0, databases and retention
+policies are mapped to buckets using the **database and retention policy (DBRP) mapping service**.
+
+The DBRP mapping service uses the **database** and **retention policy** specified in
+[compatibility API](/v2.0/reference/api/influxdb-1x/) requests to route operations to a bucket.
+[Create DBRP mappings automatically](#dbrp-mapping-behavior) with the `/write` compatibility endpoint or
+[create DBRP mappings manually](#manually-create-and-manage-dbrp-mappings) using the `/api/v2/dbrps` API endpoint.
+
+- [DBRP mapping behavior](#dbrp-mapping-behavior)
+- [Manually create and manage DBRP mappings](#manually-create-and-manage-dbrp-mappings)
+
+### Default retention policies
+A database can have multiple retention policies with one set as default.
+If no retention policy is specified in a query or write request, InfluxDB uses
+the default retention policy for the specified database.
+
+## DBRP mapping behavior
+
+- [When writing data](#when-writing-data)
+- [When querying data](#when-querying-data)
+
+### When writing data
+When writing data to InfluxDB 2.0 using the [`/write` compatibility endpoint](/v2.0/reference/api/influxdb-1x/write/),
+the DBRP mapping service checks for a bucket mapped to the database and retention policy:
+
+- If a mapped bucket is found, data is written to the bucket.
+- If an unmapped bucket with a name matching:
```

Comment:
I'm assuming that the token used MUST not be a "scoped" token??  I would expect a "scoped" token to fail to create a new bucket.  So, either the token must have all read/write access or an All Access Token??  We probably want to confirm and highlight this...

Reply:
Good point. It will need to be an All Access token.

Reply:
An important troubleshooting point here is that if you ALREADY have created an all access token, it will not have the dbrp service permissions as this is something new that has been added.  So you will need to create a new all access token and check to ensure that it has the dbrp service permissions.

Reply:
@brettbuddin can you confirm, but my understanding is the dbrp permissions are no longer required and should no longer show up in the token.

Reply:
@stuartcarnie Yeah we should probably take them out of the tokens. Nothing is relying on them directly anymore.